keyboards/jj50/keymaps/abstractkb/keymap.c

```diff
@@ -0,0 +1,139 @@
+/*
+Base Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Modified 2017 Andrew Novak <ndrw.nvk@gmail.com>
+Modified 2018 Wayne Jones (WarmCatUK) <waynekjones@gmail.com>
+Modified 2019 AbstractKB
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public LicensezZZ
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+#include "rgblight.h"
+
+#define ______ KC_TRNS
+#define _DEFLT 0
```

Comment:
**This is just an FYI:**

You could replace Lines 26-29 with an enum, which would make it easier to manage adding/removing/rearranging your layers moving forward.

This is programmatically equivalent to your code:

```c
enum layers {
    _DEFLT,
    _RAISE,
    _LOWER,
    _FN
};